vortex-array/src/arrays/varbinview/mod.rs

```diff
@@ -120,12 +120,31 @@ assert_eq_align!(BinaryView, u128);
 impl BinaryView {
     pub const MAX_INLINED_SIZE: usize = 12;
 
+    /// Create a view from a value, block and offset
+    ///
+    /// Depending on the length of the provided value either a new inlined
+    /// or a reference view will be constructed.
+    pub fn make_view(value: &[u8], block: u32, offset: u32) -> Self {
```

Comment:
Inline? Maybe worth triggering benchmakrs

Reply:
there's only one place we use it and it will show up in micro benchmarks

Reply:
ah, I didn't replace the arrow invocations. TBH arrow does something very funky with enumerating all the constant lengths since then llvm produces better code

Reply:
will actually replicate this https://github.com/apache/arrow-rs/blob/main/arrow-array/src/builder/generic_bytes_view_builder.rs#L524 remember reading the blog post and we did recreate this behaviour in... zig

Reply:
Can you leave a source comment / link in code! This is funky indeed